qiskit/quantum_info/operators/symplectic/pauli_list.py

```diff
@@ -261,6 +263,17 @@ def z(self):
     def z(self, val):
         self._z[:] = val
 
+    @property
+    def array(self):
+        """Boolean array for x and z"""
+        return np.column_stack((self.x, self.z))
+
+    @array.setter
+    def array(self, val):
+        num_qubits = val.shape[1] // 2
+        self._x[:] = val[:, 0:num_qubits]
```

Comment:
You can call the `x` setter here:
```suggestion
        self.x = val[:, 0:self.num_qubits]
```